Gemfile

```diff
@@ -60,6 +60,7 @@ gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sshkey",                         "~>1.8.0",       :require => false
+gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
Note, we have very similar issues with providers.  Even if you never use google or openstack, we install the dependencies for them.  I'm not saying we shouldn't solve that problem, it's just not that easy.  I'm very open to  suggestions.